index.js

```diff
@@ -831,6 +831,29 @@
            return maybe.isJust ? f(x, maybe.value) : x;
          });
 
+  //# Maybe#sequence :: Applicative f => Maybe (f a) ~> (a -> f a) -> f (Maybe a)
+  //.
+  //. Evaluates an applicative action contained within the Maybe, resulting in:
+  //.
+  //.   - a pure applicative of a Nothing if `this` is a Nothing; otherwise
+  //.
+  //.   - an applicative of Just the value of the evaluated action.
+  //.
+  //. ```javascript
+  //. > S.Nothing().sequence(S.Either.of)
+  //. Right(Nothing())
+  //.
+  //. > S.Just(Right(42)).sequence(S.Either.of)
+  //. Right(Just(42))
+  //. ```
+  Maybe.prototype.sequence =
+  method('Maybe#sequence',
+         {a: [Apply], b: [Apply]},
+         [$Maybe(a), $.Function, b],
```

Comment:
I don't think so. We're quite a bit beyond the city limits of sanctuary-def at this point. ;)